features/support/examples/rspec/spec_helper_only_rack_test.rb.example

```diff
@@ -3,7 +3,6 @@ require "rspec/core"
 require "rspec/matchers"
 require "rspec/expectations"
 require "raml-rb"
-require "rack/test"
 
 path = File.expand_path('../../lib', __FILE__)
 
```

Comment:
In also of rails app they now have "rails_helper" I think what we should do is create a rambo_helper that then the user can adjust to either require "spec_helper" or "rails_helper"